src/Components/Pagination.tsx

```diff
@@ -0,0 +1,33 @@
+export function Pagination({
+  usersPerPage,
+  totalUsers,
+  paginate,
+}: {
+  usersPerPage: number;
+  totalUsers: number;
+  paginate: any;
```

Comment:
pelo seu código, esse paginate seria do tipo 
```suggestion
  paginate: (pageNumber: number) => void;
```
e o que acha de alterar o nome de `paginate` para `onPageTap`?